stackoverflow-dark.css

```diff
@@ -88,8 +88,8 @@
 
   /* === background color === */
   /* from darkest to lightest grey */
-  .search-container input[type=text] {
-    background-color: #000 !important;
+  .search-container input[type=text], .topbar.newheader .topbar-wrapper .topbar-links .search-container #search input[name="q"] { 
+    background-color: #222 !important;
```

Comment:
Hi @Zeeex!

Thanks for your contribution!

Why change the background color here? Is there a reason why some of these same definitions are repeated in line 174 below?

And why am I not seeing the "newheader" on Stack overflow? Is there an opt-in somewhere to enable it? Thanks!